### PR TITLE
Insecure endpoint

### PR DESCRIPTION
**Pull request by kdihalas**
Some changes that will allow the use of terraform provider internally for development.

### CODE REVIEW

File: sdk/compute/compute_client.go [hunk 1]
```diff
@@ -26,7 +26,17 @@ type Client struct {
...
 // NewComputeClient creates a new, unauthenticated compute Client.
-func NewComputeClient(identityDomain, userName, password string, apiEndpoint *url.URL) *Client {
+func NewComputeClient(identityDomain, userName, password string, apiEndpoint *url.URL, allowInsecureEndpoint bool) *Client {
+	cfg := &tls.Config{
+		InsecureSkipVerify: false,
+	}
+
+	if allowInsecureEndpoint == true {
+		cfg = &tls.Config{
+			InsecureSkipVerify: false,
```
Comment (kdihalas) [inline sdk/compute/compute_client.go]: This is older commit i did it they way you suggest

**Comment by scross01** [comment]
@kdihalas the support for insecure endpoints is being added to the built-in provider for opc - would you be able to try https://github.com/hashicorp/terraform-provider-opc/pull/13

**Comment by scross01** [comment]
@kdihalas do you still need the insecure endpoint enhancement to be supported for this older provider.  The newer official built-in `opc` provider https://github.com/terraform-providers/terraform-provider-opc has support for the `insecure` option https://www.terraform.io/docs/providers/opc/index.html#insecure

**Comment by kdihalas** [comment]
To be honest i don't know. I am not part of Oracle anymore but since there is an official driver there is no need for this to be merged.

**Pull request closed without merging**